### PR TITLE
Add change log for 5.18.0

### PR DESCRIPTION
**Pull request by knabar**
Added #433, #435, #441 to changelog.

Also added upgrade notes to `UPGRADING.md` (formerly `MIGRATION_TO_DJANGO_32_GUIDE.md`)

### CODE REVIEW

**Comment by sbesson** [comment]
What is the current plan for the documentation of the dependency upgrade. https://github.com/ome/omero-web/pull/433#issuecomment-1386858733 suggested a few options including using the changelog, adding a migration guide similar to https://github.com/ome/omero-web/blob/master/MIGRATION_TO_DJANGO_32_GUIDE.md

**Comment by jburel** [comment]
Thanks @sbesson for the reminder. A migration doc might be easier  to find and reference
@knabar what do you think?

**Comment by knabar** [comment]
@jburel @sbesson I renamed the existing `MIGRATION_TO_DJANGO_32_GUIDE.md` to `UPGRADING.md` and expanded it with notes about this release.

**Comment by jburel** [comment]
The renaming of the file might break some links. I will double-check

**Comment by knabar** [comment]
> The renaming of the file might break some links. I will double-check

Thank you, I did not think of that.  Hopefully, if there are links, they are to a specific tag/commit and not to `HEAD`.

**Comment by jburel** [comment]
See https://github.com/knabar/omero-web/pull/1

**Comment by jburel** [comment]
* Link in https://raw.githubusercontent.com/ome/www.openmicroscopy.org/master/_posts/2022-03-21-omero-web-5.14.0.md points to the tag so we have fine on that front

**Comment by jburel** [comment]
@knabar I forgot to mention in ther release doc that for "key" release like this one we will need to add a post to the website
i will update the documentation

**Comment by jburel** [comment]
@knabar I fixed the link in the README in https://github.com/knabar/omero-web/pull/1

File: UPGRADING.md [hunk 1]
```diff
@@ -1,6 +1,44 @@
...
+
+### Connector storage in Django sessions
+
+To prepare for upcoming Django upgrades, this upgrade changes the use of
+`request.session["connector"]` from storing a Connector object to a 
+"persist and rehydration" strategy similar to how the Django authentication
+middleware handles model objects. 
+
+Any downstream implementation which was directly assigning and/or retrieving 
+`request.session["connector"]` will need to update their codebase.
```
Comment (sbesson) [inline UPGRADING.md:17]: Given there are only 2 API methods, thoughts about being explicit about the updates in the migration guide e.g.

```
- all usages of `request.session.get("connector")` should be replaced by `Connector.from_session(request)`
- all usages of `request.session["connector"] = connector` should be replaced by `connector.to_session(request)`
```

Comment (jburel) [reply UPGRADING.md:17]: It will make things clearer for dev

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
+5.18.0 (February 2023)
+----------------------
+
+## Features
+-
```
Comment (will-moore) [inline CHANGELOG.md:5]: extra - on empty line

**Review by will-moore — APPROVED** [review]

**Comment by knabar** [comment]
Added #436 to changelog

**Pull request merged**